### PR TITLE
Fix grammar

### PR DESCRIPTION
**Pull request by remitbri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 ## Why
 
 - Pulls in remote git commits to ensure you publish the latest commit
-- Reinstalls dependencies to ensure it works with the latest dependency tree
+- Reinstalls dependencies to ensure npmpub works with the latest dependency tree
```
Comment (MoOx) [inline README.md]: nope here, it's "your project", not "npmpub".

**Comment by remitbri** [comment]
(new commit / PR) Ah, ok, I was confused by the "it" anyway :)

**Comment by MoOx** [comment]
:+1:

**Comment by MoOx** [comment]
Pushed to the original repo as well :)
https://github.com/sindresorhus/np/pull/5

**Pull request merged**